src/assets/images/pause.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="23" height="25" version="1.1" xml:space="preserve"><path d="M0 0L0 25 8 25 8 0 0 0M15 0L15 25 23 25 23 0 15 0Z" fill="#368AA2"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="23" height="25" version="1.1" xml:space="preserve"><path d="M0 0L0 25 8 25 8 0 0 0M15 0L15 25 23 25 23 0 15 0Z" fill="#FFFFFF"/></svg>
```

Comment:
Always wondered why the pause button was blue.  I thought maybe it was designed to make me feel calm and want to keep listening all the way through the episode.